changelog/8587.bugfix.md

```diff
@@ -0,0 +1 @@
+Catch AttributeError in YamlStoryReader method when intent value is missing in stories or rules and re-raise as RasaException.
```

Comment:
optional: Users probably don't know the `YAMLStoryReaderr` so I'd not expose the information to them and make it a bit easier to understand.
```suggestion
Catch `AttributeError` when reading stories in YAML format when intent value is missing in stories or rules and re-raise as `RasaException`.
```

Reply:
should this really be a `misc` changelog as it doesn't really affect the users?